.github/workflows/zz_generated.create_release.yaml

```diff
@@ -0,0 +1,182 @@
+# DO NOT EDIT. Generated with:
+#
+#    devctl gen workflows
+#
+name: Create Release
+on:
+  push:
+    branches:
+      - 'legacy'
+      - 'master'
+      - 'release-v*.*.x'
+      # "!" negates previous positive patterns so it has to be at the end.
+      - '!release-v*.x.x'
+jobs:
+  debug_info:
+    name: Debug info
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Print github context JSON
+        run: |
+          cat <<EOF
+          ${{ toJson(github) }}
+          EOF
+  gather_facts:
+    name: Gather facts
+    runs-on: ubuntu-18.04
+    outputs:
+      project_go_path: ${{ steps.get_project_go_path.outputs.path }}
+      version: ${{ steps.get_version.outputs.version }}
+    steps:
+      - name: Get version
+        id: get_version
+        run: |
+          title="$(echo "${{ github.event.head_commit.message }}" | head -n 1 -)"
+          # Matches strings like:
+          #
+          #   - "release v1.2.3"
+          #   - "release v1.2.3-r4"
+          #   - "release v1.2.3 (#56)"
+          #   - "release v1.2.3-r4 (#56)"
+          #
+          # And outputs version part (1.2.3).
+          if echo $title | grep -qE '^release v[0-9]+\.[0-9]+\.[0-9]+([.-][^ .-][^ ]*)?( \(#[0-9]+\))?$' ; then
+            version=$(echo $title | cut -d ' ' -f 2)
+          fi
+          version="${version#v}" # Strip "v" prefix.
+          echo "version=\"$version\""
+          echo "::set-output name=version::${version}"
+      - name: Checkout code
+        if: ${{ steps.get_version.outputs.version != '' }}
+        uses: actions/checkout@v2
+      - name: Get project.go path
+        id: get_project_go_path
+        if: ${{ steps.get_version.outputs.version != '' }}
+        run: |
+          path='./pkg/project/project.go'
+          if [[ ! -f $path ]] ; then
+            path=''
+          fi
+          echo "path=\"$path\""
+          echo "::set-output name=path::${path}"
+  install_semver:
+    name: Install semver
+    runs-on: ubuntu-18.04
+    env:
+      BINARY: "semver"
+      URL: "https://raw.githubusercontent.com/fsaintjacques/semver-tool/3.0.0/src/semver"
+    steps:
+      - name: Key
+        id: key
+        run: |
+          cache_dir="/opt/cache"
+          cache_key="install-${BINARY}-${URL}"
+          echo "::set-output name=binary::${BINARY}"
+          echo "::set-output name=cache_dir::${cache_dir}"
+          echo "::set-output name=cache_key::${cache_key}"
+          echo "::set-output name=url::${URL}"
+      - name: Cache
+        id: cache
+        uses: actions/cache@v1
+        with:
+          key: "${{ steps.key.outputs.cache_key }}"
+          path: "${{ steps.key.outputs.cache_dir }}"
+      - name: Download
+        if: ${{ steps.cache.outputs.cache-hit != 'true' }}
+        run: |
+          # TODO check hash
```

Comment:
What does that mean?

Reply:
For security reasons I don't feel comfortable downloading third-party binary from a tag. I have a card for this in my board.